api.planx.uk/server.ts

```diff
@@ -112,7 +112,8 @@ assert(process.env.UNIFORM_SUBMISSION_URL);
 // needed for storing original URL to redirect to in login flow
 app.use(
   cookieSession({
-    maxAge: 24 * 60 * 60 * 100,
```

Comment:
value was previously set at 2.4hrs but I think was *supposed* to be 24hrs (assuming `100` was intended as `1000`), so new value is only a slight reduction, and also clearer to read

Reply:
Good catch!